backoffice/src/Controllers/AccountController.cs

```diff
@@ -12,22 +13,26 @@ public class AccountController : BaseController
     private readonly ILogger<AccountController> _logger;
     private readonly IDroitService _droitService;
     private readonly IAccountService _accountService;
+    private readonly ILetterService _letterService;
 
-    public AccountController(ILogger<AccountController> logger, IDroitService droitService, IAccountService accountService)
+    public AccountController(ILogger<AccountController> logger, IDroitService droitService, IAccountService accountService, ILetterService letterService)
     {
         _logger = logger;
         _droitService = droitService;
         _accountService = accountService;
+        _letterService = letterService;
     }
 
     
     public async Task<IActionResult> Index(SearchOptions searchOptions)
     {
         searchOptions.IncludeAssociations = true;
         searchOptions.FilterByAssignedUser = true;
+        var currentUserId = _accountService.GetCurrentUserId();
```

Comment:
You have the accountService in the letterService, better to do this there, and pass it to the repo, rather than in the controller and passing it twice